### PR TITLE
Upgrade scala compiler silencer to 1.6.0

### PR DESCRIPTION
**Pull request by oliverse-da**
Sending this out to have ci check whether increasing the silencer plugin version causes any other issues. fyi - @andreaslochbihler-da 

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [X] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [X] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: bazel-java-deps.bzl [hunk 1]
```diff
@@ -24,8 +24,8 @@ def install_java_deps():
             "com.fasterxml.jackson.module:jackson-module-parameter-names:2.9.9",
             "com.fasterxml.jackson.module:jackson-module-scala_2.12:2.9.9",
             "com.github.ben-manes.caffeine:caffeine:2.8.0",
-            "com.github.ghik:silencer-lib_2.12:1.3.1",
-            "com.github.ghik:silencer-plugin_2.12:1.3.1",
+            "com.github.ghik:silencer-lib_2.12.8:1.6.0",
+            "com.github.ghik:silencer-plugin_2.12.8:1.6.0",
```
Comment (SamirTalwar-DA) [inline bazel-java-deps.bzl:28]: [I believe we're at 2.12.11 now.](https://github.com/digital-asset/daml/pull/5807)

Comment (oliverse-da) [reply bazel-java-deps.bzl]: Trying out 2.12.11 scala version to see if that works as smoothly as 2.12.8 did on ci.

Comment (oliverse-da) [reply bazel-java-deps.bzl]: Looks like it's working. Or now @SamirTalwar-DA ?

**Comment by leo-da** [comment]
what is the difference between
```
        "@maven//:com_github_ghik_silencer_plugin_2_12",
```
and
```
        "@maven//:com_github_ghik_silencer_plugin_2_12_11",
```
I can see how this easily goes out of sync with the Scala version configured in the bazel WORKSPACE:
https://github.com/digital-asset/daml/blob/51d96a7d2e5c4305c53a32a42101f875a61da2cf/WORKSPACE#L615

Can we at least add a comment to the WORSPACE file?

**Comment by S11001001** [comment]
@leo-da See ghik/silencer#31, ghik/silencer#37.

**Comment by leo-da** [comment]
> @leo-da See [ghik/silencer#31](https://github.com/ghik/silencer/issues/31), [ghik/silencer#37](https://github.com/ghik/silencer/issues/37).

> Since 1.4.3 the plugin is fully cross versioned.
so... this means we don't have to specify the minor version at all and can just keep it at `"@maven//:com_github_ghik_silencer_plugin_2_12"`.

**Pull request merged**